3.0/json/discriminators.json

```
@@ -305,6 +305,38 @@
         }
       }
     },
+    "/redocly-discriminator-madness": {
```

```suggestion
    "/redocly-flavored-discriminator": {
```

take it or leave it!